### PR TITLE
Update members.blade.php

### PR DESCRIPTION
**Pull request by Gryph667**
Adding banner maybe

### CODE REVIEW

File: resources/views/division/members.blade.php [hunk 1]
```diff
@@ -19,7 +19,8 @@
         @endif
     </div>
     <div class="col">
-        <!-- TODO: division banner? -->
+             <img class="banner" src="{{ asset('static/img/batt_banners/' . $batt->divalias . '.png') }}"
+             alt="{{ $div->name }} Banner" title="{{ $div->name }} Banner"
```
Comment (LeoVerto) [inline resources/views/division/members.blade.php:23]: Unclosed img tag.

**Review by LeoVerto — CHANGES_REQUESTED** [review]
Same problem as #66, in fact it would probably be nicer to move these changes over to that one.

File: resources/views/division/members.blade.php (hunk 1 above)
Comment (Gryph667) [reply resources/views/division/members.blade.php]: Changed both, committed to branch.

**Comment by Gryph667** [comment]
Merging

**Pull request merged**